forum/qa-plugin/report-reason/q2apro-flag-reasons-lang-default.php

```diff
@@ -38,5 +38,7 @@
         'REPORTED_ANSWER_NOT_FOUND' => 'Nie znaleziono odpowiedzi, do której odnosi się zgłoszenie.',
         'REPORTED_COMMENT_NOT_FOUND' => 'Nie znaleziono komentarza, do którego odnosi się zgłoszenie.',
         'PAGE_NEEDS_RELOAD' => 'Zgłoszenie posta nie powiodło się.<br>Strona wymaga odświeżenia.',
+        'POST_HIDDEN' => 'Nie można zgłosić wybranego posta, ponieważ został ukryty.',
```

Comment:
Czy nie lepiej zamiast POST_HIDDEN dać np. POST_UNAVAILABLE, tak jak to ma miejsce w sytuacji, gdy ktoś pisze odpowiedź na już nieistniejący post i dostaje komunikat, że post nie jest już dostępny (ukryty lub usunięty)? Bo chyba niemożność flagowania dotyczy zarówno postów ukrytych jak i usuniętych?

Analogicznie zmieniłbym treść komunikatu na taką, która sygnalizuje niedostępność posta, a nie konkretnie jego ukrycie.

Reply:
usunięcia posta nie dodawałem, bo to już gdzieś jest obsługiwane. Spróbuj zgłosić usunięty post, u mnie pokazuje się "Nie można zgłosić tej treści". W sumie takiego samego komunikatu można też użyć do ukrycia chyba?

Reply:
Ok, ten komunikat jest obsługiwany przez core'ową metodę `qa_flag_error_html`.

Nie wiem czy w takim razie jest sens, żeby komunikat dla ukrytego postu miał generyczną treść - wersja, którą zrobiłeś jasno wskazuje, że nie można zgłosić **ukrytego** postu. A skoro core obsługuje przypadek próby zgłoszenia **usuniętego** postu, to myślę że nie ma sensu tego ruszać.

Reply:
tego przy usunięciu nie, ale skoro tam już taki jest, to może warto zmienić ten o ukrytym na taki sam. Zasadniczo użytkownika nie interesuje co się z nim stało, a raczej nie powinno, tylko to że nie może go już zgłosić, bo z jakiegoś powodu jest niedostępny obecnie

Reply:
Może i masz rację. :) Jeśli dla usuniętego posta core'owa walidacja zwraca generyczny komunikat, to dla spójności komunikat z "naszej" walidacji też może być generyczny.

Reply:
zmieniłem